src/Core/Box.cs

```diff
@@ -0,0 +1,42 @@
+
+namespace Bearded.Utilities
+{
+    /// <summary>
+    /// Static Box class to help with box creation.
+    /// </summary>
+    public static class Box
+    {
+        /// <summary>
+        /// Gets a new box witha  given value.
+        /// </summary>
+        /// <param name="value">The value of the box.</param>
+        public static Box<T> This<T>(T value)
```

Comment:
Nope, intended usage is:

`var t = Box.This(DateTime.Now);`

Do you think I should turn it into an extension method (would be valid on all value types).
Or should I rename it to `From` or something else instead?